c++/src/capnp/compat/http-over-capnp.c++

```diff
@@ -731,26 +789,64 @@ public:
     KJ_REQUIRE(!responseSent, "already called send() or acceptWebSocket()");
     responseSent = true;
 
-    auto req = clientContext.startWebSocketRequest();
+    if (headers.isWebSocket() && headers.get(kj::HttpHeaderId::SEC_WEBSOCKET_EXTENSIONS) == kj::none) {
```

Comment:
The diff ended up being really annoying, not sure if there's a clean way to fix it.
The old logic should remain as-is in the `else` clause of this statement.

Reply:
It looks fine if I tell github to hide whitespace changes.